octoprint_mrbeam/static/js/app/view-models/working-area.js

```diff
@@ -4611,6 +4614,37 @@ $(function () {
         self._sendAnalytics = function (event, payload) {
             self.analytics.send_frontend_event(event, payload);
         };
+
+        self.changeLaserCutterMode = function (selectObject) {
+            console.log('Changing laser cutter mode to', selectObject.value)
+
+            showConfirmationDialog({
+                title: gettext("Do you want to switch the mode?"),
+                message: gettext(
+                    "Keep in mind that the device will restart after switching the laser cutter mode."
+                ),
+                question: gettext(`Do you want to switch to ${selectObject.value} mode?`),
+                proceed: gettext("Continue"),
+                proceedClass: "primary",
+                cancel: gettext("Cancel"),
+                onproceed: function () {
+                    OctoPrint.simpleApiCommand("mrbeam", "lasercutter_mode_change", {"mode": selectObject.value})
```

Comment:
Just because the editor marks lasercutter as a misspell (technically they are right as it is 2 words)
```suggestion
                    OctoPrint.simpleApiCommand("mrbeam", "laser_cutter_mode_change", {"mode": selectObject.value})
```